Apps/Common/src/AspNetConfiguration/StartupConfiguration.cs

```diff
@@ -509,9 +608,11 @@ public void UseHttp(IApplicationBuilder app)
             else
             {
                 // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
+                this.Logger.LogInformation("Using HSTS, which sets Strict-Transport-Security Header");
```

Comment:
Why did you remove this logging?  It only prints on startup?

Reply:
looks to me that it was added instead of removed